src/mon/MonCommands.h

```diff
@@ -378,6 +378,7 @@ COMMAND("fs set "
         "|allow_new_snaps|inline_data|cluster_down|allow_dirfrags|balancer"
         "|standby_count_wanted|session_timeout|session_autoclose"
         "|allow_standby_replay|down|joinable|min_compat_client "
+	"|refuse_client_session|max_xattr_size "
```

Comment:
`refuse_client_session` is wrong here.

Reply:
@rishabh-d-dave pinging on this now that https://github.com/ceph/ceph/pull/50981#issuecomment-1846278730 is resolved.

Reply:
I think this has been taken care @rishabh-d-dave ?

Good to mark the comment as resolved?